Tests/webcomponents/js/ez-universal-discovery.js

```diff
@@ -0,0 +1,440 @@
+            });
+
+            it('should be false by default', function () {
+                assert.isFalse(
+                    universalDiscovery.multiple
+                );
+            });
+        });
+    });
+
+    describe('YUI stack loading', function () {
+        let renderViewSpy;
+
+        function assertViewRendered(spy, Y) {
+            assert.isOk(
+                spy.callCount
+            );
+            assert.equal(
+                spy.firstCall.args[0],
+                Y.eZ.UniversalDiscoveryView,
+                'The UniversalDiscoveryView should have been rendered'
+            );
+        }
+
+        afterEach(function () {
+            delete window.eZ;
+            renderViewSpy.restore();
+        });
+
+        describe('`ez-yui-app:ready` event', function () {
+            beforeEach(function () {
+                universalDiscovery = fixture('BasicTestFixture');
+                defineGlobals();
+                renderViewSpy = sinon.spy(window.eZ.YUI.app, 'renderSideView');
+            });
+
+            it('should render the universal discovery', function () {
+                document.dispatchEvent(new CustomEvent('ez:yui-app:ready'));
+
+                assertViewRendered(renderViewSpy, window.eZ.YUI.Y);
+            });
+        });
+
+        describe('already loaded', function () {
+            beforeEach(function () {
+                defineGlobals();
+                renderViewSpy = sinon.spy(window.eZ.YUI.app, 'renderSideView');
+                universalDiscovery = fixture('BasicTestFixture');
+            });
+
+            it('should render the universal discovery right away', function () {
+                assertViewRendered(renderViewSpy, window.eZ.YUI.Y);
+            });
+        });
+    });
+
+    describe('YUI Universal Discovery Widget', function () {
+        let renderViewStub;
+        const container = document.createElement('div');
+        const locationObject = {};
+        const contentObject = {};
+        const contentTypeObject = {};
+
+        const location = getLocationMock();
+        const content = getContentMock();
+        const contentType = getContentTypeMock();
+
+        function getLocationMock() {
+            const Model = function () {
+                this.toObject = function () {
+                    return locationObject;
+                };
+            };
+
+            return new Model();
+        }
+
+        function getContentMock() {
+            const Model = function () {
+                this.toObject = function () {
+                    return contentObject;
+                };
+            };
+
+            return new Model();
+        }
+
+        function getContentTypeMock() {
+            const Model = function () {
+                this.toObject = function () {
+                    return contentTypeObject;
+                };
+            };
+
+            return new Model();
+        }
+
+        function assertEventParam(e) {
```

Comment:
this is a very generic name. I would call it `assertEventSelection` and move the assertion on `e.bubbles` out of it